types/node/crypto.d.ts

```diff
@@ -991,6 +1011,13 @@ declare module "crypto" {
         options?: CipherGCMOptions,
     ): DecipherGCM;
     function createDecipheriv(
+        algorithm: CipherChaCha20Poly1305Types,
+        key: CipherKey,
+        iv: BinaryLike,
+        options?: CipherChaCha20Poly1305Options,
+    ): DecipherChaCha20Poly1305;
+    // eslint-disable-next-line @definitelytyped/no-unnecessary-generics
+    function createDecipheriv<T extends string>(
```

Comment:
The orphaned type parameters (and eslint pragmas) can be removed on `createDecipher` and `createDecipheriv`, otherwise LGTM!

Reply:
Done.